src/controllers/bookingController.js

```diff
@@ -0,0 +1,60 @@
+/* eslint-disable eqeqeq */
+/* eslint-disable class-methods-use-this */
+
+import Bookings from '../services/bookingServices';
+import Response from '../utils/response';
+import requestService from '../services/requestService';
+
+/** Class representing room booking. */
+class RoomBooking {
+  /**
+   * Creates a new accommodation.
+   * @param {object} req request
+   * @param {object} res response
+   * @param {object} next next
+   * @returns {object} accommodation object
+   */
+  async bookRooms(req, res, next) {
+    try {
+      const books = [];
+      for (let count = 0; count < req.body.rooms.length; count += 1) {
```

Comment:
what do u think if we change from books to bookings? As it represents bookings made to room.